remoulade/actor.py

```diff
@@ -195,8 +220,16 @@ def message_with_options(self, *, args=None, kwargs=None, **options) -> Message:
         for middleware in self.broker.middleware:
             options = middleware.update_options_before_create_message(options, self.broker, self.actor_name)
 
+        if queue_name is not None:
+            if queue_name != self.queue_name and (
```

Comment:
add a `queue_names` property and use it here `queue_name not in self.queue_names`  and in the broker init